internal/bridge/crunchybridgecluster/delete.go

```diff
@@ -28,7 +28,7 @@ func (r *CrunchyBridgeClusterReconciler) handleDelete(
 	log := ctrl.LoggerFrom(ctx)
 
 	// If the CrunchyBridgeCluster isn't being deleted, add the finalizer
-	if crunchybridgecluster.ObjectMeta.DeletionTimestamp.IsZero() {
```

Comment:
:question: Here and throughout, is the suggestion about removing `ObjectMeta` tied to a particular version of the API?